.travis.yml

```diff
@@ -11,5 +11,5 @@ matrix:
       - beta
 script:
   - cargo build --verbose --all
-  - cargo test --verbose --all
+  - cargo test --verbose
```

Comment:
The `noconst` feature is currently necessary for `gen_ecmult` and `gen_genmult`, but if it's enabled, then tests won't work, because nearly all user-facing functions depend on those constants. In the future, switching to build script will fix this.